string language
string[] words
string[] grammar
string[] sentences
---
string error
